lib/activerecord-bitemporal/bitemporal.rb

```diff
@@ -432,9 +432,9 @@ def destroy(force_delete: false)
         current_time = Time.current
         target_datetime = valid_datetime || current_time
 
-        with_transaction_returning_status do
-          duplicated_instance = self.class.find_at_time(target_datetime, self.id).dup
+        duplicated_instance = self.class.find_at_time(target_datetime, self.id).dup
 
+        ActiveRecord::Base.transaction(requires_new: true) do
```

Comment:
Isn't `joinable: false` necessary?
https://makandracards.com/makandra/42885-nested-activerecord-transaction-pitfalls

Reply:
Thanks :)
I fixed it https://github.com/kufu/activerecord-bitemporal/pull/77/commits/f80bb2ba0e9ad322d6aea52f76d7d039333f886f